### ISSUE TITLE
indent bug with braceless for loops

### ISSUE THREAD

**Issue by feross** [eslint/eslint#3291]
Using eslint 1.0.0 and this config:

```
"indent": [2, 2, { "SwitchCase": 1 }]
```

This code:

```js
for (let a of [])
  for (let b of [])
    for (let c of [])
      console.log('hi')
```

Gives these unexpected errors:

```
t.js:3:5: Expected indentation of 2 characters but found 4.
t.js:4:7: Expected indentation of 4 characters but found 6.
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by gyandeeps** [comment]
Duplicate #3238
It has been fixed but not yet released.
